### ISSUE TITLE
feature request time :-)

### ISSUE THREAD

**Issue by Wuerfel21** [MightyPirates/OpenComputers#260]
After tinkering aroumd with OC i see its one of the best (maybe the best of all) but there aresome feature reqeusts:

A forth tier, whose microchips you could get from a special villager(you cant make such epic stuff yourself, you need an expert) with a trade like this(super expensive): tier 3 chip + 20 emeralds = tier 4 chip. All the tier 4 stuff could have this enchanted effect (like a enchanted pickaxe).

So, a tier 4 screen could have 320x100 pixels with full 24-bit colors.
A tier 4 cpu could do 32 components(a server would do 128 with 4xtier 4 cpu) 
A tier 4 gpu could get 4x tier 3 speed.
A tier 4 hdd could have 8 mb.
And so on

Also the villager should buy and sell other computer items(most tier 1,some tier 2, rarely tier 3,and a 1/1000 chance tier 4)

I would also like to have tiered floppys(which can all be read with the same drive) and tiered remotes

And you should make the new unrafter: expensive(4 diamonds or so) and inefficient(cant uncraft chips(too high precision stuff) and dont get evrything back). Cause otherwise you dont need to gather resources anymore.

That was it, :-)

**Comment by fnuecke** [comment]
Getting some villager interaction may be interesting, but I'll admit I'm not a fan of resources that cannot be - at least to some degree or in some fashion - be automatically harvested.

As for a tier 4:
- For screens / GPUs that's not realistic, I'm afraid, for reasons mentioned in previous tickets with that request (in short: network limitations).
- For CPUs ... if someone shows me a (reasonable!) setup where they need more than 256 components connected to one machine (T3 server) we can talk again :-)
- For RAM... the current maximum is as far as I personally feel comfortable going, to be honest. I'd be interested in other voices on this, though. The main reason for this limitation is of course using the mod on servers, where this avoids a single user eating up all the (real) RAM of the server using one computer.
- HDDs... I don't see a real need to go much bigger than what's already possible. If you need more space, get creative. Build more servers, link them together using the network and write a script that provides an abstracted, multi-server NFS.
- I might add a 'creative' CPU, RAM, ... that are really really big.

If you have a look [here](http://oc.cil.li/index.php?/topic/82-heads-up-on-opencomputers-13/) you'll be happy to see that a Disassembler is already planned (and implemented, in fact, just not pushed, yet).

So, in short: I think T3 stuff is powerful enough as it is, but I'll think of creating creative tier parts to go with the creative case.

Thanks for the feedback!

**Comment by LordFokas** [comment]
As a CS Engineering student, I have to support Sangar here. If you can farm resources hard enough to get to that super powerful Tier IV, there is no need to get creative with your limited resources.

That is the beauty of computers and engineering in general: to come up with elegant solutions using limited resources to solve your problems. Like distributing a massive file system over a server cluster and providing access abstraction so that all that hardware seems to behave like a hard drive.

**Comment by Wuerfel21** [comment]
Thanks anyway!
With the network problems we could send the actual function called to theclient and let the client calculate.
Development sneak-peek:NoteAL almost complete, some evil software-pirate program and some webbrowser-like piece of junk are queued!!!
